### ISSUE TITLE
Blur previous focus while Yippy open

### ISSUE THREAD

**Issue by msaspence** [mattDavo/Yippy#11]
When you open Yippy the previous input remains focused. This sometime leads to me forgetting that Yippy is even open, and I end up typing something in there and then when Yippy steals the space key event I wonder what is going on for a second. It would be much smoother if the focus was completely taken by Yippy while it was open.

Presumably this would need to happen if search was implemented anyway.

**Comment by mattDavo** [comment]
Yeah this is another valid concern, but my first intention. I designed Yippy to initially fit my need of wanting a fast access stack of recent copies. Mainly for when I code so I move chunks around very quickly. E.g. very quickly opening the window and paste something recent straight in with very few key presses.

When I first built this I didn't have much experience with macOS development so this was the best solution I could find. However, since then I have seen it done in other ways and think it should be possible to get the solution we want:
- Taking focus
- Still able to auto paste for us
- Search

But unfortunately this is fair chunk of work, and don't have time right now, but I will get to it! 😄

**Comment by karlhorky** [comment]
I like the feature of Yippy not taking focus! (this is also a feature of other clipboard managers such as [Maccy](https://github.com/p0deje/Maccy) and [Flycut](https://github.com/TermiT/Flycut).

One small tweak to improve the UX of the unfocused window may be to dismiss it (same dismissal as done when <kbd>esc</kbd> is pressed) when something else is interacted with (eg. user clicks on the background or cmd-tabs).

This may make the unfocused behavior easier to live with, at least temporarily.

What are your thoughts @mattDavo ?

**Comment by paul-blum** [comment]
> I like the feature of Yippy not taking focus! (this is also a feature of other clipboard managers such as [Maccy](https://github.com/p0deje/Maccy) and [Flycut](https://github.com/TermiT/Flycut).
> 
> One small tweak to improve the UX of the unfocused window may be to dismiss it (same dismissal as done when esc is pressed) when something else is interacted with (eg. user clicks on the background or cmd-tabs).
> 
> This may make the unfocused behavior easier to live with, at least temporarily.
> 
> What are your thoughts @mattDavo ?

Yes that would be awesome! At the moment it is a bit obstrusive.
